pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,36 +1,36 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
```

Comment:
That depends on all the fixes going in and if enough map to UWP scenarios to warrant rolling them up.  With `5.2.2` we had to ship it due to the NuGet package layout changes (`nestandard`), nothing in packaging is forcing that this time around, so it's more a discussion of release management.